atomic_reactor/plugins/hide_files.py

```diff
@@ -55,26 +61,46 @@ def run(self):
             for file_to_hide in files_to_hide
         ]
 
-        hide_in_build_dir = functools.partial(self._add_hide_lines, start_lines, end_lines)
+        hide_in_build_dir = functools.partial(self._add_hide_lines, custom_image_index,
+                                              start_lines, end_lines)
         self.workflow.build_dir.for_each_platform(hide_in_build_dir)
 
+    def _get_custom_image_index(self) -> List[int]:
+        """get indexes for baseimage scratch stages"""
+        custom_image_index = []
+        df_images = self.workflow.data.dockerfile_images.original_parents
+
+        self.log.info("df images: '%s'", df_images)
+
+        for idx, img in enumerate(reversed(df_images)):
+            self.log.info("parent_image '%s' : '%s'", idx, str(img))
```

Comment:
is this important info to log, or log at info level?

Reply:
and we have three log messages in this function

Reply:
they were just for debugging removing them